frontend/components/Profile/Profile.jsx

```diff
@@ -0,0 +1,280 @@
+import React from 'react';
+import { Divider, Button } from '@blueprintjs/core';
+import PropTypes from 'prop-types';
+import gql from 'graphql-tag';
+import { Mutation } from 'react-apollo';
+import InputField from '../commons/InputField';
+import FormContainer from '../FormContainer';
+import useFormInput from '../hooks/useFormInput';
+import samePassword from '../../lib/samePassword';
+
+const UPDATE_ME = gql`
```

Comment:
Personal preference: I like having a`queries` folder that contains all my GraphQL queries, so if I want to call them from different components in the future, I only have to change them in one place. Also a nice way of separating concerns